Functions.Templates/Templates/SqlInputBinding-CSharp/readme.md

```diff
@@ -0,0 +1,7 @@
+# SQL Input Binding - C<span>#</span>
+
+The `SQL Input Binding` makes it easy to take a SQL query to run on a provided object and returns the output of the query
```

Comment:
Not sure "provided object" is the terminology we want to use here - I'd probably just stick with table/view. @dzsquared ?

Reply:
The `SQL Input Binding` makes it easy to retrieve data from a database, returning the output of the query or stored procedure to the function.